slave.mk

```diff
@@ -158,6 +158,10 @@ export DEFAULT_CONTAINER_REGISTRY
 export MIRROR_URLS
 export MIRROR_SECURITY_URLS
 
+ifeq ($(SONIC_SMARTSWITCH),1)
+SMARTSWITCH = 1
```

Comment:
>SMARTSWITCH [](http://example.com/codeflow?start=0&length=11)

rules/config file already define the macro, should we not change it?

Reply:
slave.mk is useing the same pattern for other macros.